<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMapEnumerateSettingToolBarWidget</class>
 <widget class="QWidget" name="ColorMapEnumerateSettingToolBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QCheckBox" name="visibleCheckBox">
     <property name="text">
      <string>Color Bar</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="horizontalRadioButton">
     <property name="text">
      <string>Horizontal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="verticalRadioButton">
     <property name="text">
      <string>Vertical</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>visibleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visibleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>verticalRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
